### DISCUSSION TITLE
How can I merge presets "Nerd Font Symbols" & "Bracketed Segments"?

### DISCUSSION THREAD

**Discussion by liblaf** [starship/starship#4939] [Q&A] [answered]
As described, I wish to use preset [Bracketed Segments](https://starship.rs/presets/bracketed-segments.html) with Nerd Font support.

**Comment by mickimnet** [comment, marked as answer]
You can do a `starship preset bracketed-segments --output starship-bracketed-segments.toml`,  `starship preset nerd-font-symbols --output starship-nerd-font-symbols.toml` and then merge them either with a tool of your choice or do it manually.
